src/Game/Command/StartCommand.php

```diff
@@ -28,6 +32,10 @@ public function __construct(RealTimeClient $client, GameManager $gameManager, Me
         if ($this->channel[0] == 'D') {
             throw new Exception("Can't start a game by direct message.");
         }
+
+        //TODO: This should probably be refactored to be passed within constructor or refactored to allow for per game settings. Leaving here until decision is made on what to do with this.
```

Comment:
The strategy should probably be configurable per game, yeah. 

Utilizing dependency injection is always more testable and favorable in general but I didn't write any of the code this way or write any unit tests because i am a Bad Person so not gonna enforce this or anything haha :P